tracee/tracee.bpf.c

```diff
@@ -194,6 +209,7 @@ typedef struct context {
     u32 eventid;
     s64 retval;
     u8 argnum;
+    char stack_id[STACK_ID_LEN];
```

Comment:
This can be u32 instead (the return value of bpf_get_stackid is int, and you ignore negative values)

Reply:
Sorry, this one is currently holding this review up.

I 100% agree that this should be a `uint32` and not a char array, but *something* is truncating the uint32 when unmarshalling it, and converting only the first 'byte' of the int. Looking into this atm, I'm sure it's an issue with my code, somewhere

Reply:
Ok, worked it out - `uint32`s also want to be aligned on a boundary (makes sense), which the 1-byte `argnum` throws this off, resulting in the 'padding' being in between `argum` and `stackid`.

I think the correct solution here is to keep the 1-Byte `argnum` parameter always at the end of the struct, and just put `stackID` before it (and make sure the struct is the same on the Golang and C sides). I've also added a note a bout this in a comment, in case the next person to alter the context struct has the same issue.